src/style.css

```diff
@@ -0,0 +1,149 @@
+@font-face {
+  src: url(./fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-weight: bold;
+}
+
+body {
+  font-weight: 300;
+  margin: 0 8px;
+  box-sizing: content-box;
+
+}
+
+form {
+
+  width: 100%;
+  box-sizing: content-box;
+}
+
+.search_bar {
+  width: 100%;
+  margin: 20px 0;
+  position: relative;
+}
+
+.search_bar_little {
+  width: 100%;
+  position: relative;
+}
+
+.search_bar input {
+  width: 100%;
+  height: 70px;
+  padding: 27px 0 26px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search_bar_little input {
+
+  width: 100%;
+  margin-bottom: 0;
+  padding: 11px 0;
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 2px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search_bar input::placeholder {
+
+  font-family: Avenir, sans-serif;

```

Comment:
add font-family for html and use inherit here
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit